.github/workflows/reusable-pytest.yml

```
@@ -7,7 +7,7 @@ on:
       python_versions:
         required: false
         default: >-
-          ["3.8", "3.9", "3.10"]
+          ["3.8", "3.9", "3.10", "3.12"]
```

```suggestion
          ["3.9", "3.10", "3.11", "3.12"]
```
I think we can follow Numpy and the Scientific Python ecosystem and drop 3.8 at this point. There's also no reason to skip a version